### PR TITLE
Deprecating Web SQL deprecation OT

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- It now starts in 117.

### CODE REVIEW

**Review by rachelandrew — APPROVED** [review]

File: site/en/blog/web-sql-deprecation-timeline-updated/index.md [hunk 1]
```diff
@@ -44,6 +44,6 @@ removal in insecure contexts and eventually all contexts is planned for Chromium
 - [🔮 In the future.] A
   [deprecation trial](/docs/web-platform/origin-trials/#deprecation-trials) to
   keep using Web SQL is available from **Chromium&nbsp;118**
```
Comment (hansifer) [inline site/en/blog/web-sql-deprecation-timeline-updated/index.md:46]: should this be `**Chromium&nbsp;117**` ?

Comment (tomayac) [reply site/en/blog/web-sql-deprecation-timeline-updated/index.md:46]: Oh, good catch. I'll open a quick follow-up PR.

Comment (tomayac) [reply site/en/blog/web-sql-deprecation-timeline-updated/index.md:46]: https://github.com/GoogleChrome/developer.chrome.com/pull/6726

Comment (hansifer) [reply site/en/blog/web-sql-deprecation-timeline-updated/index.md:46]: Thanks!

Am I correct in understanding that the "ChromeDate` for 117 is also when the deprecation trial registration will become available at https://developer.chrome.com/origintrials/#/trials/active or will that be earlier?

Comment (tomayac) [reply site/en/blog/web-sql-deprecation-timeline-updated/index.md:46]: That’s my understanding too.

**Pull request merged**